TOOLING.md

```diff
@@ -0,0 +1,15 @@
+# Tooling
+
+This documents describes at a high level some the key tools used across `near-js`.
+
+## Package Manager - [PNPM](https://pnpm.io/)
+PNPM is a package manager focussed on fast installations while still be disk efficient. It manages dependency installations across the monorepo as well as linking interdependent packages. Unlike other package managers, PNPM has a non-flat `node_modules` structure which ensures packages only have access to dependencies defined in their `package.json`.
```

Comment:
> while still be

should be `while still being`

also
> non-flat

maybe "nested"? I get that it's to contrast from NPM's flat structure but it just reads a little odd

Reply:
It's the same terminology PNPM uses on their website 🤔 How about "does not flatten dependencies within `node_modules`?

Reply:
If that's how they describe it then I think best to leave it as is 👍